package.json

```diff
@@ -14,7 +14,7 @@
     "lint:fix": "yarn lint --fix",
     "create-modular-react-app": "ts-node packages/create-modular-react-app/src/cli.ts",
     "modular": "ts-node packages/modular-scripts/src/cli.ts",
-    "test": "cross-env NODE_OPTIONS=--max_old_space_size=5120 yarn modular test --regex --watchAll false --runInBand --env node --transformIgnorePatterns \"node_modules/(?!rollup)/\"",
+    "test": "cross-env NODE_OPTIONS=--max_old_space_size=5120 yarn modular test --swc --runInBand --env=node",
```

Comment:
Over time between test bugs and trying to make things work I must have messed up our default test command - should be alright now